### PR TITLE
Add new workflows to upgradeWorkflowComponents.py for HLT

### PR DESCRIPTION
**Pull request by chayanit**
This is PR to add new upgrade workflows as requested by HLT (STEAM) https://its.cern.ch/jira/browse/PDMVRELVALS-115 and Phase2 L1T https://its.cern.ch/jira/browse/PDMVRELVALS-116. All GEN fragments have been reviewing in #33409 , #33414 , #33433 and #33386. They need for Run3 validation campaign in master release and special request for Phase2 L1T.

Need for 11_3_0 so the backport is needed

### CODE REVIEW

**Comment by chayanit** [comment]
please test

**Comment by chayanit** [comment]
+1

**Comment by chayanit** [comment]
please test

This PR can be merged only after all GEN fragments are merged or finalised

**Comment by chayanit** [comment]
please test

File: Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py [hunk 1]
```diff
@@ -1413,4 +1413,13 @@ def __init__(self, howMuch, dataset):
...
     ('GluGluTo2Jets_M_300_2000_14TeV_Exhume_cff',UpgradeFragment(Kby(9,100),'GluGluTo2Jets_14TeV')),
+    ('TTbarToDilepton_mt172p5_TuneCP5_14TeV_pythia8_cfi',UpgradeFragment(Kby(9,50),'TTbarToDilepton_14TeV')),
+    ('QQToHToTauTau_mh125_TuneCP5_14TeV_pythia8_cfi',UpgradeFragment(Kby(9,50),'QQToHToTauTau_14TeV')),
+    ('ZpToEE_m6000_TuneCP5_14TeV_pythia8_cfi',UpgradeFragment(Kby(9,50),'ZpToEE_m6000_14TeV')),
+    ('ZpToMM_m6000_TuneCP5_14TeV_pythia8_cfi',UpgradeFragment(Kby(9,50),'ZpToMM_m6000_14TeV')),
+    ('SMS-T1tttt_mGl-1500_mLSP-100_TuneCP5_14TeV_pythia8_cfi',UpgradeFragment(Kby(9,50),'SMS-T1tttt_14TeV')),
+    ('VBFHZZ4Nu_TuneCP5_14TeV_pythia8_cfi',UpgradeFragment(Kby(9,50),'VBFHZZ4Nu_14TeV')),
+    ('EtaBToJpsiJpsi_14TeV_TuneCP5_pythia8_cfi',UpgradeFragment(Kby(9,50),'EtaBToJpsiJpsi_14TeV')),
+    ('WToLNu_14TeV_TuneCP5_pythia8_cfi',UpgradeFragment(Kby(21,50),'WToLNu_14TeV')),
+    ('WprimeToLNu_M2000_14TeV_TuneCP5_pythia8_cfi',UpgradeFragment(Kby(21,50),'WprimeToLNu_M2000_14TeV')),
```
Comment (srimanob) [inline Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py:1424]: I don't see this GEN fragment in
https://github.com/cms-sw/cmssw/tree/master/Configuration/Generator/python

Comment (chayanit) [reply Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py:1424]: Hi @srimanob they are all on the way
https://github.com/cms-sw/cmssw/pull/33407

**Comment by chayanit** [comment]
please test

**Comment by chayanit** [comment]
+1

I would need this PR to be merged for final 11_3_0

**Comment by srimanob** [comment]
> I would need this PR to be merged for final 11_3_0

But this means you will need to make a backport PR, I assume.

**Comment by srimanob** [comment]
+Upgrade

This PR is to add configuration in runTheMatrix. It needs to be merged together with https://github.com/cms-sw/cmssw/pull/33407. Backport PR to 11_3_0 to be done.

**Comment by chayanit** [comment]
Uh oh I don't know how to do the backport

**Comment by srimanob** [comment]
> Uh oh I don't know how to do the backport. What about all GEN fragments?

GEN fragment backport is in place, https://github.com/cms-sw/cmssw/pull/33596

How to make backport:
https://cms-sw.github.io/tutorial-resolve-conflicts.html
(looking for Backporting a PR).

**Comment by chayanit** [comment]
There are a lot more to be backported  #33409 , #33414 , #33433 and #33386 :(

**Comment by qliphy** [comment]
+1

**Pull request merged**